app/src/main/java/org/wikipedia/edit/Edit.kt

```diff
@@ -0,0 +1,74 @@
+package org.wikipedia.edit
+
+import org.wikipedia.dataclient.mwapi.MwPostResponse
+
+class Edit : MwPostResponse() {
+    private val edit: Result? = null
+    fun edit(): Result? {
+        return edit
+    }
+
+    fun hasEditResult(): Boolean {
+        return edit != null
+    }
+
+    class Result {
+        private val result: String? = null
+        private val newrevid: Long = 0
+        private val captcha: Captcha? = null
+        private val code: String? = null
+        private val info: String? = null
+        private val warning: String? = null
+        private val spamblacklist: String? = null
+        fun status(): String? {
```

Comment:
Most of these getter functions no longer need to exist. (kotlin automatically generates getter functions for public fields)